pyseq.py

```diff
@@ -408,6 +429,57 @@ def append(self, item):
         else:
             raise SequenceError('Item is not a member of this sequence')
 
+    def get_max_mtime(self):
+        '''
+        returns the latest mtime of all items
+        '''
+        maxDate = list()
+        for i in self:
+            maxDate.append(i.mtime)
+        return max(maxDate)
+    
+    def get_size(self):
+        '''
+        returns the size all items 
+        divide the result by 1024/1024 to get megabytes
+        '''
+        tempSize = list() 
+        for i in self:
+            tempSize.append(i.size)
+        return sum(tempSize)
+    
+    def reIndex(self, addSub, padding=None, run=False):
```

Comment:
If you're increasing the frame numbers, couldn't this possibly overwrite files?  Adding 5 on a range of 1001-1010 would move 1001 to 1006 before 1006 has been moved.
I'd recommend reversing your for loop if the addSub is > 0, and keeping it like it is if addSub < 0